lumicks/pylake/tests/data/mock_confocal.py

```diff
@@ -119,6 +155,123 @@ def generate_photon_count_line(line):
     )
 
 
+def confocal_mock_file_h5(
```

Comment:
I'm curious for which tests you intend to use `confocal_mock_file_h5` versus `generate_kymo_with_ref` vs just generating a simple `Kymo` from an image directly with `_kymo_from_array`.

I imagine the last is still preferred for many downstream tasks (since it is much more performant than actually churning out an `infowave`) unless you are specifically testing properties that explicitly test reconstruction, reshaping or interaction with other channels.

It might make sense to detail the use cases for each of them somewhere so we have some guidelines for when the code is less fresh in all our memories.